app/assets/javascripts/templates/bulk_buy_modal.html.haml

```diff
@@ -14,20 +14,20 @@
       {{ "js.bulk_buy_modal.min_quantity" | t }}
     %div
       %button.bulk-buy-add.variant-quantity{type: "button", ng: {click: "add(-1)", disabled: "!canAdd(-1)"}}>
-        {{ "js.shop_variant_no_group_buy.decrement" | t }}
+        －
```

Comment:
I agree with this change. Can you just add a comment here making it obvious this is a special character and not a normal minus sign?
Maybe what was on the en.yml file?
－ # U+FF0D Fullwidth Hyphen-Minus


＋ # U+FF0B Fullwidth Plus Sign

Reply:
Done.